src/status_im/contexts/profile/settings/view.cljs

```diff
@@ -9,18 +9,24 @@
             [status-im.contexts.profile.settings.list-items :as settings.items]
             [status-im.contexts.profile.settings.style :as style]
             [status-im.contexts.profile.utils :as profile.utils]
+            [status-im.feature-flags :as ff]
             [utils.debounce :as debounce]
             [utils.i18n :as i18n]
             [utils.re-frame :as rf]))
 
+(defn show-settings-item?
+  [{:keys [feature-flag]}]
+  (or (ff/enabled? feature-flag)
+      (nil? feature-flag)))
+
 (defn- settings-item-view
   [data]
   [rf/delay-render
    [quo/category
     {:list-type       :settings
      :container-style {:padding-bottom 12}
      :blur?           true
-     :data            data}]])
+     :data            (filterv show-settings-item? data)}]])
```

Comment:
Why do we need the collection to be a vector?

Reply:
I'm not sure if it needs to be vector, but it happens to be a vector already, and it seems safer to maintain that choice since it could affect how the components access the data. For example, the `peek` function behaves differently on vectors (looks at the last item) vs lists/seqs (looks at first item).

Reply:
Do we use `peek`?

Reply:
Well I'm not sure if we use `peek` atm. Though I would say the main idea is that from the point of view of the data (where the data is defined), the data is a vector, and unless we have a good reason to change it from a vector to something else, we should try to maintain that decision for downstream functions/components. 

That way if there are functions/components that want to use `peek` or something, we can safely look at the definition of the data to figure out what the semantics will be when using `peek`.

Reply:
Just reading your comment now @ilmotta 👀 
 
If quo components don't really care about things being vectors then I'll change this to be normal `filter`.
Thanks for your thoughts @ibrkhalil and @ilmotta 🙏